tests/z3/z3_test.cpp

```diff
@@ -6,36 +6,169 @@
 
 #include "z3_factory.h"
 
+#include "z3_factory.h"
+
 using namespace smt;
 using namespace std;
+//using namespace z3;
+
+void printVec(string name, TermVec ts) {
```

Comment:
nitpicky and not that big of a deal, but could you call it `print_vec`? Just trying to use the underscore syntax for all functions even if it's in a test.